packages/react/src/themes/teams/components/Icon/iconStyles.ts

```diff
@@ -130,6 +130,10 @@ const iconStyles: ComponentSlotStylesInput<IconProps, IconVariables> = {
       ...(!rtl && {
         transform: `rotate(${rotate}deg)`,
       }),
+      // override base theme default rtl behavior
+      ...(rtl && {
```

Comment:
intent of this and the previous object expansion expressions could be provided in a simpler way:

```jsx
transfrom: rtl ? 'unset' : `rotate(${rotate}deg)`
```